### ISSUE TITLE
Missing exports condition

### ISSUE THREAD

**Issue by BCsabaEngine** [vnphanquang/svelte-put#266]
SvelteKit said that missing svelte export condition when installing/using svelte-put/modal.
https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition

svelte-put/modal: 1.0.5
@sveltejs/kit": "2.0.6",

**Comment by vnphanquang** [comment]
@BCsabaEngine thank you for reporting. Which vite version are you using? And is it a warning or an error that blocks you from using the package? Thanks

**Comment by BCsabaEngine** [comment]
It does not blocking work, just a warning at npm install and every build. My vite is the latest 5.0.10?

**Comment by BCsabaEngine** [comment]
diff

```
diff --git a/node_modules/@svelte-put/modal/package.json b/node_modules/@svelte-put/modal/package.json
index b7f5941..f73fe59 100644
--- a/node_modules/@svelte-put/modal/package.json
+++ b/node_modules/@svelte-put/modal/package.json
@@ -11,7 +11,9 @@
     "./Modal.svelte": "./dist/Modal.svelte",
     "./ModalPortal.svelte": "./dist/ModalPortal.svelte",
     "./modal": "./dist/modal.js",
-    ".": "./dist/index.js"
+    ".": {
+      "svelte": "./dist/index.js"
+    }
   },
   "types": "dist/index.d.ts",
   "typesVersions": {
```

**Comment by BCsabaEngine** [comment]
Simply just remove ```"svelte": "./dist/index.js"```, export format is ok.

so the necessary diff is:
```
diff --git a/node_modules/@svelte-put/modal/package.json b/node_modules/@svelte-put/modal/package.json
index b7f5941..62d8fc3 100644
--- a/node_modules/@svelte-put/modal/package.json
+++ b/node_modules/@svelte-put/modal/package.json
@@ -3,7 +3,6 @@
   "version": "1.0.5",
   "description": "Svelte async modal",
   "type": "module",
-  "svelte": "./dist/index.js",
   "main": "./dist/index.js",
   "module": "./dist/index.js",
   "exports": {
```

**Comment by BCsabaEngine** [comment]
Can I help yout with a PR?

**Comment by vnphanquang** [comment]
Thanks @BCsabaEngine for the investigation and offer to help. Let me take some time to do that today to save you the trouble of forking/pr :pray:

**Comment by vnphanquang** [comment]
Should be resolved in [@svelte-put/modal@1.0.6](https://github.com/vnphanquang/svelte-put/releases/tag/%40svelte-put%2Fmodal%401.0.6). Thank you again for your help @BCsabaEngine :data:
